util/bklog/log.go

```diff
@@ -37,6 +43,8 @@ func GetLogger(ctx context.Context) (l *logrus.Entry) {
 
 	if logger != nil {
 		l = logger.(*logrus.Entry)
+	} else if logger := log.GetLogger(ctx); logger != nil {
+		l = logger
```

Comment:
may overwrites `log.L = L` in `init` too, to make sure to use same fallback.

https://github.com/containerd/containerd/blob/main/log/context.go#L65

Reply:
Thank you for the review. Fixed this.